app/scripts/modules/ui/SplitContainer.js

```diff
@@ -1,5 +1,7 @@
 'use strict';
 
+require('./datagrid/UIUtils.js');
```

Comment:
we added the UIUtils.js helper into the "datagrid" folder. Now need we import it into a more generic control. It is OK as is, but just wondered if it makes sense to move UIUtils.js to the upper level folder (outside "datagrid") as well... what do you think?